nixos/modules/services/misc/paperless.nix

```diff
@@ -18,17 +18,13 @@ let
 in
 {
   options.services.paperless = {
-    enable = mkOption {
-      type = lib.types.bool;
-      default = false;
-      description = ''
-        Enable Paperless.
+    enable = mkEnableOption ''
```

Comment:
`mkEnableOption` does

https://github.com/NixOS/nixpkgs/blob/14ee8fefd7e5cb11d3e429e48a8f1c71f7ff9be9/lib/options.nix#L68

So it's not really right to use it for this.

Reply:
Is [this](https://github.com/NixOS/nixpkgs/pull/44086/commits/d0ed820adaeb6a022f3f746b999e8e8f5ebde3b1) okay, or too tricky?

Reply:
Well `mkEnableOption` really just wants to take a `name` and there's a lot more text here that isn't part of the name.

Reply:
Alright, ditched it.